### DISCUSSION TITLE
Param to hide page from listing?

### DISCUSSION THREAD

**Discussion by patrickbenkoetter** [gohugo-ananke/ananke#777] [2 - Q&A / Support]
Greetings,

I'm fairly new to the world of hugo and more than that to ananke so please forgive me if I'm asking a question that more educated people know.

Here's the problem I'm trying to solve:
I need to add a privacy statement and an imprint to my blog by law. Both pages need to be accessibly with only one click from each page.

I've managed to create the pages and add them to the menu. However the pages show up in the blog list and I haven't found a way to hide them from the list of (new) posts.

My reading suggests to add a condition that checks if e.b. a param `list` is set to `hide` or `show` and list the post only if it would say show. Is that the way to go with ananke? Is there a generic hugo way and I haven't found it yet?

TIA

Patrick

**Comment by davidsneighbour** [comment]
You came across one of the features that is missing in Ananke. I will add it relatively soon, but until then you should be able to fix this reading through [the mainSections documentation](https://gohugo.io/methods/site/mainsections/). Basically if you define "post" as main section (if you followed the content setup of the example site) all other pages should be hidden from these menus/lists. The problem is, that to make this properly work we will need to rename the content folder to `posts` (plural) --- as far as I understand it, some testing is required.

**Reply by patrickbenkoetter** [reply]
I tried that, but succeeded only partially. Quite likely due to my still limited hugo skills. But I'm fine for the moment and I can wait until you come up with the new feature.

**Comment by ramiro** [comment]
Could the Hugo `build.list` front matter option be useful in this case? https://gohugo.io/content-management/build-options/

**Reply by patrickbenkoetter** [reply]
If I do that and set

```
build:
  list: never
```

the page is not in the listing, but also the reference in the menu is gone. Back to drawing board…

**Reply by davidsneighbour** [reply]
I would have tried

```
[build]
  list = 'local'
```

That should IMHO result in the file not being included in the global .RegularPages, but the ones from the section the file is in. The issue here is probably with your content structure. Is your `about.md` inside of `content` or inside of `content/anotherfolder`?
